### PR TITLE
PRESIDECMS-345 : Make user group as required for admin users.

### PR DESCRIPTION
**Pull request by cfmitrah**

### CODE REVIEW

File: system/forms/preside-objects/security_user/admin.edit.self.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@ This form is used for the "edit user" form in the user admin section of the admi
 	<tab id="basic">
 		<fieldset id="basic">
 			<field name="active" deleted="true" />
+			<field binding="security_user.groups" required="false"/>
```
Comment (DominicWatson) [inline system/forms/preside-objects/security_user/admin.edit.self.xml]: A user should not be able to edit their own groups :) Please remove this line.

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Looking good, but please see review comment.

File: system/forms/preside-objects/security_user/admin.edit.self.xml (hunk 1 above)
Comment (cfmitrah) [reply system/forms/preside-objects/security_user/admin.edit.self.xml]: Removed edit functionality for user group

**Comment by DominicWatson** [comment]
Nice one, thanks for this. Merged into the upcoming 10.8.0 release.

**Pull request merged**